managing_providers/_topics/adding_amazon_cloud_regions.md

```diff
@@ -0,0 +1,33 @@
+# Adding Amazon Cloud Regions
+
+{{ site.data.product.title_short }} allows administrators to add Amazon cloud regions on
+the appliance server. You can use this capability to add new regions that have been
+set up since {{ site.data.product.title_short }} was released.  Once adding the region it will be
+available when creating a new Amazon EC2 provider.
+
+1.  Click ![config gear](../images/config-gear.png) (**Configuration**).
+
+2.  Click on the **Settings** accordion, then click **Zones**.
+
+3.  Click the zone where the {{ site.data.product.title_short }} server is located,
+    then click on the EVM server.
```

Comment:
Maybe we should make it a region setting?

Reply:
I mirrored the settings from "disabling a region"  If we change it here we should change it there also, you want to change in both spots?

Reply:
Done

Reply:
yeah i think in both places makes sense.